src/igvfd/audit/file.py

```diff
@@ -115,3 +115,47 @@ def audit_external_reference_files(value, system):
                 f'but does not have identifier(s) from an external resource listed in `dbxrefs`.'
             )
             yield AuditFailure(audit_message.get('audit_category', ''), f'{detail} {audit_message.get("audit_description", "")}', level=audit_message.get('audit_level', ''))
+
+
+@audit_checker('AlignmentFile', frame='object')
+def audit_bai_alignment_files(value, system):
+    '''
+    [
+        {
+            "audit_description": "Alignment files in bai format are expected to have their corresponding bam files in `derived_from`.",
```

Comment:
Here I would say "bam file" (singular) to indicate we are expecting exactly one bam.